### ISSUE TITLE
windows-386 binary is not being produced on deploy

### ISSUE THREAD

**Issue by daviddias** [ipfs/distributions#165]
dist.ipfs.io stopped shipping windows-386 binaries and after running the make step in my machine, here is the build log:

```
# gx/ipfs/Qme3aYm74r4gyMjtZTXpopHkBA6qjU21LXJy4GF56pLkW8/badger/y
../gopath/src/gx/ipfs/Qme3aYm74r4gyMjtZTXpopHkBA6qjU21LXJy4GF56pLkW8/badger/y/mmap_windows.go:70:12: constant 4294967295 overflows int
../gopath/src/gx/ipfs/Qme3aYm74r4gyMjtZTXpopHkBA6qjU21LXJy4GF56pLkW8/badger/y/mmap_windows.go:70:12: array bound is too large
```

Is this known? Was there a decision of not shipping windows-386 binaries anymore? 

I'll be waiting for an answer so that I can do the release dance for js land. ref: https://github.com/ipfs/npm-go-ipfs-dep/pull/21#issuecomment-348466417

**Comment by Stebalien** [comment]
https://github.com/ipfs/go-ipfs/issues/4438

**Comment by Stebalien** [comment]
New tests so this doesn't happen again: https://github.com/ipfs/go-ipfs/pull/4445

**Comment by diasdavid** [comment]
@Stebalien will this be shipped in 0.4.14 only? There is a bunch of CI that fails because appveyor uses the windows-386 version.

**Comment by Stebalien** [comment]
It'll be shipped in all future versions. go-ipfs 0.4.13 simply doesn't build on 386 windows.
